README.md

```diff
@@ -0,0 +1,29 @@
+# Red Hat–PyCon CZ Coding Challenge
+
+We have prepared 3 programming problems for you to solve so you can win a prize and maybe even a job at Red Hat!
+
```

Comment:
To be more straightforward I would add something like:
"As a first step, clone this repository with `git clone git@github.com:fedora-python/pyconcz-challenge.git`"